Makefile

```diff
@@ -84,6 +84,9 @@ all: test forklift-controller
 # Run tests
 test: generate fmt vet manifests validation-test
 	go test -coverprofile=cover.out ./pkg/... ./cmd/...
+	go test ./virt-v2v/cold/...
```

Comment:
`go test -coverprofile=cover.out ./pkg/... ./cmd/... ./virt-v2v/...`
or with `cold` path if you wish to be more specific

Reply:
@liranr23 added.